tests/test_computational_resources.py

```diff
@@ -572,9 +572,9 @@ def test_resource_setup_widget_for_password_configure(monkeypatch, tmp_path):
 
     # Test select a new resource setup will update the output interface (e.g. ssh_config, computer_setup, code_setup)
     # and the computer/code setup widget will be updated accordingly.
-    w.comp_resources_database.domain_selector.value = "merlin.psi.ch"
```

Comment:
This was super-painful to fix (these tests now fail on master as well). Really these tests should be rewritten to use mocked-out data instead of real data that can change at any moment.

Reply:
See also this note. 

https://github.com/aiidalab/aiidalab-widgets-base/pull/698/files#diff-71fe007c958acd11b69e1011aaaa2116a3a7f62a2e5ef8ec3743a8d191dcec88R49